packages/shopify-app-session-storage-redis/src/migrations.ts

```diff
@@ -1,21 +1,22 @@
-import {createClient} from 'redis';
 import {Session} from '@shopify/shopify-api';
 
-type RedisClient = ReturnType<typeof createClient>;
+import {RedisConnection} from './redis-connection';
+
+export const migrationMap = new Map([
+  ['migrateToVersion1_0_1', migrateToVersion1_0_1],
```

Comment:
I dont believe we can change the version name because this has already been released and if we change it the migrator will determine that tis migration has not happened. I put a comment in the code and change the name of the function at least.